packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/app/{{project_name}}.dart

```diff
@@ -9,7 +9,9 @@ import 'package:provider/provider.dart';
 
 import '../../l10n/l10n.dart';
 import '../../lib_auth/data_sources/remote/interceptors/auth_interceptor.dart';
-import '../../lib_router/router.dart';
+import '../../lib_router/router.dart'; {{#enable_change_language}}
+import '../common_blocs/coordinator_bloc.dart';
+import '../data_sources/local/shared_preferences_instance.dart';{{/enable_change_language}}
```

Comment:
As disclosed the UI layer should not have access to the data layer

Reply:
I have deletednusing the `shared_preferences_instance`